lib/cn/kvs_mblk_desc.c

```diff
@@ -40,6 +43,7 @@ mblk_mmap(struct mpool *mp, uint64_t mbid, struct kvs_mblk_desc *md)
     md->map_base = (void *)base;
     md->alen_pages = props.mpr_alloc_cap / PAGE_SIZE;
     md->wlen_pages = props.mpr_write_len / PAGE_SIZE;
+    md->ra_pages = HSE_RA_PAGES_MAX;
```

Comment:
Going to open a ticket, we need to get the read-ahead size for the device at open time.  The default is 128K for typical 9200 and 5210 type drives, but then if you have a striped LV you multiply that by the stripe size.  This can greatly reduce the number of madvise() calls we need to make.
Currently, if the read-ahead size is less then 128K then our madvise loop will do a poor job and that leads to severe QOS degration during split which holds up spills.